features/fixtures/mazerunner/jvm-scenarios/src/main/java/com/bugsnag/android/mazerunner/scenarios/OnSendCallbackScenario.kt

```diff
@@ -14,6 +15,8 @@ internal class OnSendCallbackScenario(
     init {
         config.addOnSend(
             OnSendCallback { event ->
+                event.clearFeatureFlag("deleteMe")
+                event.addFeatureFlag(event.featureFlags[0].name, "b")
```

Comment:
Do we maybe want to use `getFeatureFlags()` as well here? Something like:

```suggestion
                event.clearFeatureFlag("deleteMe")
                if (event.getFeatureFlags().single() != FeatureFlag("fromStartup", "a")) {
                    return false
                }
                event.addFeatureFlag(event.featureFlags[0].name, "b")
```

Reply:
That's in essence what it's already doing. `event.featureFlags[0].name` calls `getFeatureFlags()`, fetches index 0, and gets its name. It then adds that feature flag's name with a new variant. If any of that fails, the test will fail.